src/routes/mint/[address]/[id].svelte

```diff
@@ -1,15 +1,141 @@
 <script lang="ts">
 	import { page } from '$app/stores';
 	import { onMount } from 'svelte';
-
+	import { env } from '$lib/constants';
+	// import { tokenUriToMetaData } from '$lib/nft-utils';
+	// import { evaluateNft, getTokenUriType, getImageType } from '$lib/utils/functions';
+	// import defaultAbi from '$lib/defaultAbi';
+	// import { Contract, ethers } from 'ethers';
+	import { user } from '$lib/stores/user';
+	
+	let errorPreparingMint = false;
+	let readyToMint = false;
+	let newTokenURI = '';
+	
+	let newMetadata = {};
+	
 	onMount(async () => {
 		// Do initialization things here
+		
+		// const node1 = await Ipfs.create();
+		// const result = await node1.add("hello world2");
+		// debugger;
```

Comment:
If you're ready to merge can we cut the commented out things here and in the imports?